Validation/SiTrackerPhase2V/python/HLTPhase2TrackerValidationFirstStep_cff.py

```diff
@@ -0,0 +1,37 @@
+import FWCore.ParameterSet.Config as cms
+from Validation.SiTrackerPhase2V.Phase2ITValidateCluster_cff import *
+from Validation.SiTrackerPhase2V.Phase2OTValidateCluster_cff import *
+from Validation.SiTrackerPhase2V.Phase2ITValidateRecHit_cff import *
+from Validation.SiTrackerPhase2V.Phase2ITValidateTrackingRecHit_cff import *
+from Validation.SiTrackerPhase2V.Phase2OTValidateTrackingRecHit_cff import *
```

Comment:
why is this file being duplicated? 
See https://github.com/cms-sw/cmssw/blob/master/Validation/SiTrackerPhase2V/python/HLTPhase2TrackerValidationFirstStep_cff.py